### ISSUE TITLE
Publishing TS to npm

### ISSUE THREAD

**Issue by nhannah** [terrysahaidak/react-native-gallery-toolkit#24]
Like the project, but currently you are publishing the TS files to npm instead of the compiled js + types.

**Comment by terrysahaidak** [comment]
Hi there, sorry, I've been using it from github directly, didn't notice any problems.

I've just published a new release `1.0.0-alpha.1.1` removing patch-package and using "module" target for react-native as a default one.

Let me know how it works for you.

**Comment by owinter86** [comment]
Hey @terrysahaidak I think the published npm is pointing to the incorrect types path in the package.json

`"types": "lib/typescript/src/index.d.ts",`

but see its correct in the master branch.

**Comment by terrysahaidak** [comment]
Yeah, I already pushed it, but haven't published. Will do in a minute. Thanks for reminding :)

**Comment by terrysahaidak** [comment]
Done. Let me know if it works.
